src/Containers/Microsoft.NET.Build.Containers/Registry/Registry.cs

```diff
@@ -215,6 +215,7 @@ async Task<ManifestV2> ReadManifest()
             {
                 manifest.KnownDigest = knownDigestValue;
             }
+            manifest.MediaType ??= initialManifestResponse.Content.Headers.ContentType!.MediaType;
```

Comment:
ContentType may not always be present - so we probably cannot forcibly unwrap the mediaType here. As long as the 'writing' side of our tools can set a default media type when creating new images, we don't need to force an unwrap of the media type here.

Reply:
`ReadManifest()` is only called once here and ContentType is not null
https://github.com/dotnet/sdk/blob/1ed097ec1a8e88f9427b540ac3f83ac132b3f767/src/Containers/Microsoft.NET.Build.Containers/Registry/Registry.cs#L181-L186